### DISCUSSION TITLE
New project without git clone?

### DISCUSSION THREAD

**Discussion by danawoodman** [Instawork/hyperview#796] [Q&A]
Is the documentation's example of cloning the repo and running the demo the only way to setup a hyperview project?

I'd love for a more standard approach where hyperview generates the expo side of things and I can setup my own backend afterwords. Right now I have to remove a lot of code to get to that desired state.

Maybe I'm missing something, but is there no equivalent to `npm create hyperview`-type generator?

**Comment by flochtililoch** [comment]
hi @danawoodman, we don't have a generator as of right now, but this came up before and is on our backlog: #451. As of right now, the best way to try out the demo is to clone the repo or DL an archive and follow the setup instructions for the demo app.
